src/components/hermes-skills/SkillDetail.tsx

```
@@ -54,6 +55,67 @@ function docsCollapsible(body: string): boolean {
+ */
+function DocsPending({ phase, skillKey }: { phase: DetailPhase; skillKey: string }) {
+  const COPY = useCopy();
+  const fetching = phase === 'docs';
+  const [elapsed, setElapsed] = useState(0);
+
+  // Keyed on the identity the FETCH uses — `identifier || id` plus the tab the
+  // pick came from — not on `skill.id`: the same string is a lock name in the
+  // Installed tab and a registry identifier in Browse, and `onOpenSkill` can
+  // swap one for the other without unmounting this view.
+  useEffect(() => {
+    setElapsed(0);
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/components/hermes-skills/SkillDetail.tsx:96:5
  94 |   // swap one for the other without unmounting this view.
  95 |   useEffect(() => {
> 96 |     setElapsed(0);
     |     ^^^^^^^^^^ Avoid calling setState() directly within an effect
  97 |     if (!fetching) return;
  98 |     const startedAt = Date.now();
  99 |     const timer = setInterval(() => {

(react-hooks/set-state-in-effect)